app/controllers/all-service-transactions/get-controller.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+// NPM dependencies
+const _ = require('lodash')
+
+const { response, renderErrorView } = require('../../utils/response')
+const { ConnectorClient } = require('../../services/clients/connector_client.js')
+const serviceService = require('../../services/service_service')
+const transactionService = require('../../services/transaction_service')
+const { buildPaymentList } = require('../../utils/transaction_view.js')
+const { getFilters, describeFilters } = require('../../utils/filters.js')
+const router = require('../../routes.js')
+const states = require('../../utils/states')
+const client = new ConnectorClient(process.env.CONNECTOR_URL)
+
+const { CORRELATION_HEADER } = require('../../utils/correlation_header.js')
+
+module.exports = async (req, res) => {
+  const correlationId = req.headers[CORRELATION_HEADER] || ''
+  const servicesRoles = _.get(req, 'user.serviceRoles', [])
+  const filters = getFilters(req)
+  try {
+    const aggregatedGatewayAccountIds = servicesRoles
+      .flatMap(servicesRole => servicesRole.service.gatewayAccountIds)
+      .reduce((accumulator, currentValue) => accumulator.concat(currentValue), [])
```

Comment:
Similarly I think this is the only unique behaviour in this controller, we can probably abstract this out -- this would also mean that the current test coverage covers these new controllers (as there is only one)